runtime/compiler/compiler.mk

```diff
@@ -54,9 +54,18 @@ ifeq ($(PLATFORM),ppc64-linux64-clang)
     endif
 endif
 
+ifeq ($(PLATFORM),s390-linux-gcc)
+     export JITSERVER_SUPPORT=1
+endif
+
+ifeq ($(PLATFORM),s390-linux64-gcc)
+    export JITSERVER_SUPPORT=1
+endif
+
 ifeq ($(PLATFORM),s390-zos-vacpp)
```

Comment:
@mpirvu @dchopra001 Will JITServer be supported on zos? If not, I'm wondering if `JITSERVER_SUPPORT` should be removed for `s390-zos-vacpp` and `s390-zos64-vacpp`. 

```
ifeq ($(PLATFORM),s390-zos-vacpp)
    export JITSERVER_SUPPORT=1
endif
ifeq ($(PLATFORM),s390-zos64-vacpp)
    export JITSERVER_SUPPORT=1
```

Reply:
There are no immediate plans to implement JITServer on any other platform beyond Linux. Last time I've heard protobuf library was not compiling on zOS, so I could safely leave zOS out for the time being.

Reply:
I have removed the checks for zOS here: 45c234d